pages/create-account.js

```diff
@@ -0,0 +1,143 @@
+import Head from 'next/head';
+import React, { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Account() {
+  const [errors, setErrors] = useState({
+    fname: '',
+    lname: '',
+    uname: '',
+    pword: '',
+    confirmpword: '',
+  });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const inputs = {};
+    const formData = new FormData(e.target);
+    formData.forEach((value, key) => (inputs[key] = value));
+    
+    // Validation criteria for each field
+    for (const input in inputs) {
+      let errorMessage;
+      switch (input) {
```

Comment:
It's common practice to add a `default` case when writing `switch` statements because it costs nothing and it's insurance against failure when updating the program.